frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/queries.ts

```diff
@@ -0,0 +1,48 @@
+import * as _ from 'lodash';
+
+export enum VMQueries {
+  CPU_USAGE = 'CPU_USAGE',
+  MEMORY_USAGE = 'MEMORY_USAGE',
+  FILESYSTEM_USAGE = 'FILESYSTEM_USAGE',
+  NETWORK_INOUT_USAGE = 'NETWORK_INOUT_USAGE',
+  // NETWORK_IN_USAGE = 'NETWORK_IN_USAGE',
+  // NETWORK_OUT_USAGE = 'NETWORK_OUT_USAGE',
+}
+
+const queries = {
```

Comment:
please check the queries for https://github.com/openshift/console/pull/3142

Reply:
Thanks. Fixed with best effort, I do not have access to a 4.3 env.